command/meta.go

```diff
@@ -119,6 +120,8 @@ func (m *Meta) FlagSet(n string, fs FlagSetFlags) *flag.FlagSet {
 	if fs&FlagSetVars != 0 {
 		f.Var((*kvflag.Flag)(&m.flagVars), "var", "")
 		f.Var((*kvflag.FlagJSON)(&m.flagVars), "var-file", "")
+		// designate any of the above variables that are private
+		f.Var((*sliceflag.StringFlag)(&m.flagPrivVars), "private", "")
```

Comment:
I think this is the wrong place to put this, since, looking at the comment, these are flags for all commands. I think this should go in `command/push.go` around line 50. On the bright side, I think this becomes simpler